blog.md

```diff
@@ -1,53 +1,62 @@
-# Docker Voluminous plugin from ClusterHQ
+# Making the (bridging the gap/making the leap/easing the transition) development to production
```

Comment:
@wallnerryan agreed, I was just putting all the ideas i had in brain for a title. @lukemarsden (discussed in office)  wants to more accurately frame this not about going towards production but more about making the development experience faster/better.